examples/ammonia_fit_example.py

```diff
@@ -1,7 +1,8 @@
 import pyspeckit
+import astropy.units as u
 
 # Grab a .fits spectrum with a legitimate header
-sp = pyspeckit.Spectrum('../pyspeckit/tests/G031.947+00.076_nh3_11_Tastar.fits')
+sp = pyspeckit.Spectrum('G031.947+00.076_nh3_11_Tastar.fits')
```

Comment:
Yes but it is still sketchy. These paths work if the examples are ran through python setup.py test, if a user tries to run a single example the paths break.